content/chat/patterns/storage.textile

```diff
@@ -0,0 +1,190 @@
+---
+title: Message storage
+meta_description: "Store your messages outside of Ably."
+product: chat
+languages:
+  - javascript
+---
+
+Use of room "history":/chat/rooms/history enables you to retrieve messages stored by Ably from between 72 hours, up to 30 days. In some instances you may wish to store messages for longer than this, or you would prefer to store all messages in your own database.
```

Comment:
Let's skip the "longer than this" narrative and only keep it about the preference to use their own DB. Perhaps we can add a CTA to raise a feature request if they'd like to enable storage for longer than 30 days.